### PR TITLE
Add: support inclusions

### LINKED ISSUE #307: Add ability to keep additional sites on themes
I have a series of themes that are suitable for more sites than those the creator of the theme specified, thus I add those sites to the ones applicable by the theme. However, whenever an update is out for the said themes, my custom added sites prevent them from being updated, or the additional sites are deleted if I want the update to proceed. It would be nice to have a way to keep the sites the user has added on each theme through the updates - after all, the user added them having the knowledge that they may not be supported one day, so should the theme not work anymore on these sites it is easy to remove them from the theme. Thank you for your work!

### PR DESCRIPTION
**Pull request by eight04**
Fixes #307

### CODE REVIEW

File: edit/settings.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function StyleSettings(editor) {
 
   function update(newStyle, reason) {
     if (!newStyle.id) return;
-    if (reason === 'editSave' || reason === 'config') return;
+    if (reason === 'editSave') return;
```
Comment (tophf) [inline edit/settings.js:30]: Why?

Comment (eight04) [reply edit/settings.js:30]: We have to refresh the setting page when style setting changes outside of the setting page. For example:

1. Install a style.
2. Open style setting page.
3. Open the popup, toggle exclusions.

You will see that the change to exclusions is also reflected on style setting page.

Should we add some comment?

Comment (tophf) [reply edit/settings.js:30]: Update the page if it's not modified or show our standard question styleUpdateDiscardChanges?

Comment (eight04) [reply edit/settings.js:30]: Note that setting page doesn't bind to the editor and the dirty system. When a setting is changed, it calls the API `API.styles.config` immediately. You can verify this by adding `*` to inclusions. The style will be applied to the page when the textarea loses focus.

Unlike editor, setting page doesn't store a "change", so there is nothing can be discarded.

----

For the editor, there is a similar situation: toggling a style. When the editor is dirty, toggling a style in the popup will also toggle the `Enabled` checkbox on the page, which modifies the dirty state. When the editor is not dirty, the checkbox is toggled without modifying the dirty state.

Comment (tophf) [reply edit/settings.js:30]: Ah, I misunderstood the change. Now I see it adds the support, not removes it.

**Pull request merged**